helm/nessie/templates/configmap.yaml

```
@@ -26,6 +26,10 @@ metadata:
     {{- if .Values.configMapLabels }}
     {{- toYaml .Values.configMapLabels | nindent 4 }}
     {{- end }}
+  annotations:
+    {{- if .Values.configMapAnnotations }}
+    {{- toYaml .Values.configMapAnnotations | nindent 4 }}
+    {{- end }}
```

Two minor suggestions:

1. Put the if block start before `annotations:`
2. Use `tpl` to templetize the input (I know we don't do this for `configMapLabels` line 27 – but maybe we should, in fact)

```suggestion
  {{- if .Values.configMapAnnotations }}
  annotations:
    {{- tpl (toYaml .Values.configMapAnnotations) . | nindent 4 }}
  {{- end }}
```

---

@adutra wanna just commit the suggestion and then merge?